plugins/inventory/foreman.py

```diff
@@ -143,7 +147,7 @@ def _get_json(self, url, ignore_errors=None, params=None):
             if params is None:
                 params = {}
             params['page'] = 1
-            params['per_page'] = 250
+            params['per_page'] = self.get_option('batch_size')
```

Comment:
I guess, you need to introduce the default value somewhere.
I cannot believe, ansible is taking that from the `DOCUMENTATION`.

Reply:
Afaik, for inventories it does

Reply:
weird!